tests/smoke_tests/test_cluster_job.py

```diff
@@ -556,6 +556,7 @@ def test_tpu_vm_pod():
 
 
 # ---------- TPU Pod Slice on GKE. ----------
+@pytest.mark.gke
```

Comment:
What does it mean by having two marks for a single test? Our current handling of the marks means that if a test function is marked with a specific cloud, it can only be run with `--cloud-name`. How can we trigger this test now?

Reply:
This is not used directly by pytest. Some test cases can run on Kubernetes after executing `sky local up`.

However, some tests require a real TPU cluster or GKE resources. I need to schedule these test cases to run on the real GKE cluster instead of the local one. I use this marker to distinguish them on buildkite.

On buildkite, some tests will run on the kubernetes cluster set up by `sky local up`, while others will run on the GKE cluster. (I was hoping all tests could run on the local Kubernetes cluster, but it seems that’s not possible.)